definitions/infra-prometheus-redis/dashboard.json

```diff
@@ -0,0 +1,130 @@
+{
+    "title": "Redis status",
+    "icon": "line-chart",
+    "grid_column_count": 12,
+    "filter": null,
+    "owner": {
+        "email": "svandamme@newrelic.com",
+        "id": 2359884
+    },
+    "widgets": [
+        {
+            "title": "Number of connections received per second",
+            "nrql": "FROM Metric SELECT average(redis.instance.net.connectionsReceivedPerSecond) SINCE 1 day ago TIMESERIES COMPARE WITH 7 days ago",
```

Comment:
I suspect that this somehow should isolate to the individual guid we'll be synthesising. Could that be something we patch post registration? Or could we make use of a placeholder? eg `where guid = ${synthesised_guid}` cc @srvaroa @IreneMLC

Reply:
The placeholder seems like a good idea to me, since I guess that we'll have to replace the GUID when the template is requested from us.

Reply:
Update on this: We've decided to add this to a post process so the template definition here will remain as is. Thanks!